en/docs/quick-starts/use-artifact-binding-sample.md

```diff
@@ -47,7 +45,6 @@ You have successfully set up SAML artifact binding. See the sections below for m
 -----
 
 !!! info "Related Topics"
-    - [Concept: SAML2 Artifact Binding](TODO:insert-link-to-concept)
-    - [Guide: Use SAML2 Artifact Binding](../../guides/login/use-artifact-binding-sample)
-    - [Guide: Enable Login for a SAML Application](../../guides/login/webapp-saml)
-    - [Guide: SAML Authentication Errors](TODO:insert-link)
```

Comment:
is this guide not available?

Reply:
No, we don't have a doc for this in 5.11.0 either